src/Tribe/Views/V2/Rest_Endpoint.php

```diff
@@ -20,6 +20,10 @@
  */
 class Rest_Endpoint {
 
+	const NONCE_ACTION = '_view_rest';
+	const NONCE_A_KEY = '_view_rest_nonce_a';
```

Comment:
Is there a more descriptive name we can give them like main/alt? 
Not sure it makes a difference.

Reply:
Good feedback.